includes/functions.php

```diff
@@ -409,7 +409,10 @@ function is_activitypub_request() {
  * @return boolean True if the post is disabled, false otherwise.
  */
 function is_post_disabled( $post ) {
-	$post       = \get_post( $post );
+	$post = \get_post( $post );
+
+	\clean_post_cache( $post );
```

Comment:
I think that this will cause load problems more than it will solve postmeta cache problems? Is this actually doing anything useful?

Eg we could clean the cache up in `Dispatcher::send_post` above, before this check gets called. But even there I have my doubts that this is solving a problem the way it should be solved.